packages/graph/hash_graph/lib/graph/src/store/postgres/query/table.rs

```diff
@@ -102,17 +103,22 @@ macro_rules! impl_ontology_column {
                         Self::OwnedById => "owned_by_id",
                         Self::UpdatedById => "updated_by_id",
                         Self::Schema(None) => "schema",
-                        Self::Schema(Some(path)) => match path {
-                            JsonField::Json(field) => {
-                                return write!(fmt, r#"."schema"->'{}'"#, field);
-                            }
-                            JsonField::Text(field) => {
-                                return write!(fmt, r#"."schema"->>'{}'"#, field);
-                            }
-                            JsonField::Parameter(index) => {
-                                return write!(fmt, r#"."schema"->>${}"#, index);
-                            }
-                        },
+                        Self::Schema(Some(path)) => {
+                            return match path {
+                                JsonField::Json(field) => {
+                                    write!(fmt, r#"."schema"->'{field}'"#)
+                                }
+                                JsonField::JsonParameter(index) => {
+                                    write!(fmt, r#"."schema"->${index}"#)
+                                }
+                                JsonField::StaticText(field) => {
+                                    write!(fmt, r#"."schema"->>'{field}'"#)
+                                }
+                                JsonField::StaticJson(field) => {
+                                    write!(fmt, r#"."schema"->'{field}'"#)
+                                }
```

Comment:
The above is a leading question for these statements specifically, would be worried of any sort of sql injection. I suppose we don't implement the access syntax and we shouldn't have any issues yet, but just want to make sure this is in mind

Reply:
I noticed that as well. Maybe we want to `panic!` on `JsonField::Json`?

Reply:
Yes, I think that would be nice :)

Reply:
We are now `panic!`ing 8abaa19018691a7c84e29f45760f7b9f16cdb823